glmmTMB/R/utils.R

```diff
@@ -213,9 +213,69 @@ nullSparseMatrix <- function() {
         do.call(Matrix::sparseMatrix, c(argList, list(repr="T")))
     }
 }
+    
+## Check for version mismatch in dependent binary packages
+#' @importFrom utils packageVersion
+checkDepPackageVersion <- function(dep_pkg="TMB",this_pkg="glmmTMB",write_file=FALSE) {
+    cur_dep_version <- as.character(packageVersion(dep_pkg))
+    fn <- sprintf("%s-version",dep_pkg)
+    if (write_file) {
```

Comment:
I think the file should be written at install time. It seems to me it never is (`write_file=FALSE`)?